BUILDING.md

```diff
@@ -0,0 +1,45 @@
+# Building the Player
+We use `grunt` and a few `npm scripts` to build the player, lint code, and run tests. Debug code is built to `/bin-debug`, while minified & uglified code is built to `/bin-release`. Code is built with `webpack`, linted with `eslint` and `flow`, and tested with `karma` and `qunit`.
+
+1. Fork the project, clone your fork, and set up the remotes:
+````
+# Clone your fork of the repo into the current directory
+git clone https://github.com/<your-username>/jwplayer
+# Navigate to the newly cloned directory
+cd jwplayer
+# Assign the original repo to a remote called "upstream"
+git remote add upstream https://github.com/jwplayer/jwplayer
+````
+
+2. Install the dependencies:
```

Comment:
@johnBartos removing the line breaks should fix that. Line numbers reset to 1 in md after a line break.